frontend/Dockerfile

```diff
@@ -16,6 +16,13 @@ RUN chmod +x entrypoint.sh
 RUN yarn build
 RUN chgrp -R 0 /app && \
     chmod -R g=u /app
+RUN rm -rf ./node_modules
+
+FROM node:18-alpine
+WORKDIR /app
+
+COPY --from=builder /app .
+RUN yarn global add sirv-cli
```

Comment:
this reduces the size of the image significantly
<img width="1213" alt="Screenshot 2024-03-27 at 21 08 29" src="https://github.com/redhat-appstudio/quality-dashboard/assets/4881144/9c59cd7e-d51b-41be-90dd-52fbb9061e80">